mailscanner/sf_version.php

```diff
@@ -93,7 +93,8 @@
     if (get_conf_var('MTA', true) === 'postfix') {
         echo '<br>' . "\n";
         echo 'Postfix ' . __('version11') . ' ';
-        passthru("/usr/sbin/postconf -d | grep 'mail_version =' | cut -d' ' -f3");
+        exec("which postconf", $postconf);
+        passthru("$postconf[0] -d | grep 'mail_version =' | cut -d' ' -f3");
```

Comment:
You shoud add a check in case the exec() above doesn't find the command
```php
if (isset($postconf[0])) {
passthru("$postconf[0] -d | grep 'mail_version =' | cut -d' ' -f3");
} else {
echo 'postconf not found';
}
```

Reply:
Yes, I could do - although in that case it will fail as it did previously and just not print the version - but you are right, it would be better to display an error rather than a blank :)

Reply:
Done.  I was just being lazy ;)